### ISSUE TITLE
On initialization, `allowDecimalPadding` option `'floats'` does not hide the decimal zeroes if set in the html attribute

### ISSUE THREAD

**Issue by camilova** [autoNumeric/autoNumeric#652]
### Current behavior
If have this options to my "decimal" inputs:

```js
decimal_autonumeric_options = {
    modifyValueOnWheel: false,
    digitGroupSeparator: '.',
    decimalCharacter: ',',
    decimalPlaces: 2,
    maximumValue: 999999999999,
    unformatOnSubmit: true,
    allowDecimalPadding: 'floats',
    decimalCharacterAlternative: ".",
    formatOnPageLoad: true,
  }
```

I initialize the inputs with:

`AutoNumeric.multiple "input.decimal", decimal_autonumeric_options `

### Expected behavior

When raw value of a input is, for example: **150**

It should show on input "**150**" on initalization and first show of the page.

But, actually shows "**150,0**"

Afer a mouseover/mouseenter event, the input becomes "150", so, Autonumeric removes the zero decimal digit after mouseenter, exactly like we wants due option allowDecimalPadding: 'floats', but it behaves wrong on init.

What a did to fix this behavior was, adding a "fake" mouseenter event after initialization to format correctly:

```js
// Fixer for decimal digits after initialization
$.each($('input.decimal'), function(name, element) {
  var autonumeric_instance;
  autonumeric_instance = AutoNumeric.getAutoNumericElement(element);
  if (autonumeric_instance) {
    return autonumeric_instance._onFocusInAndMouseEnterFunc(new Event('mouseenter'));
  }
});

```
All the inputs will show correctly after that fixer.

**¿Is this a bug or I have something wrong on my options?**

**Comment by AlexandreBonneau** [comment]
@camilova I tested under Firefox and Chromium, and couldn't reproduce the bug using the latest `4.5.7` version.

Could you please provide a reproducible test case ?

**Comment by camilova** [comment]
@AlexandreBonneau [here is an example](https://jsfiddle.net/jxLtdr80/4/), this shows "100,0" on initialization, but should be only "100". After mouseover will format correctly.

**Comment by AlexandreBonneau** [comment]
We assumed when the decimal part *value* was equal to zero, that there was no need to bother with it, and it got concatenated while still having one or more `'0'` as a string.
The assumption is now removed, and the fix is in `v4.5.8` !
